nautobot/extras/models/groups.py

```diff
@@ -90,7 +88,7 @@ def __str__(self):
         return self.name
 
     def natural_key(self):
```

Comment:
Don't we also need to change `DynamicGroupQuerySet.get_by_natural_key()`? (May be unnecessary in the long term given #3423)